src/pages/mypage/components/navBar/TapNavBar.tsx

```
@@ -0,0 +1,32 @@
+  onTabChange: (tab: 'saveditem' | 'genImg') => void;
+}
+
+const TapNavBar = ({ activeTab, onTabChange }: TapNavBarProps) => {
+  return (
+    <div className={styles.tapNavBar}>
+      <button
+        className={styles.tapButton({
+          state: activeTab === 'genImg' ? 'active' : 'inactive',
+        })}
+        onClick={() => onTabChange('genImg')}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)